apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/account/LedgerEntry.java

```diff
@@ -255,7 +255,10 @@ public boolean equals(Object obj) {
      * @throws  SQLException            Database error occurred
      */
     void save(Connection con) throws SQLException {
-        try (final PreparedStatement stmt = con.prepareStatement("INSERT INTO account_ledger " + "(account_id, event_type, event_id, holding_type, holding_id, change, balance, " + "block_id, height, timestamp) " + "VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?, ?)", Statement.RETURN_GENERATED_KEYS)) {
+        try (final PreparedStatement stmt = con.prepareStatement(
```

Comment:
Is annotation
@DatabaseSpecificDml(DmlMarker
is missing here ?

Reply:
this annotation marks only SQL that is specific to DB. INSERT is common for all RDBMS